bossggle.js

```diff
@@ -0,0 +1,28 @@
+// setup letters
+
+var playWord = "";
+
+setupLetters();
+
+setupListeners();
+
+$(document).ready(function() {
+	$("#submit").on("click", function() {
```

Comment:
we're fixing everything to be 2 spaces.

we decided that we want to keep these as ids. In a larger codebase that we were planning to extend a bunch, I can see the value of using classes, but I like using ids to help me read and parse the html.